example/rcnn/symdata/loader.py

```diff
@@ -129,7 +129,7 @@ def getpad(self):
 
 class AnchorLoader(mx.io.DataIter):
     def __init__(self, roidb, batch_size, short, max_size, mean, std,
-                 feat_sym, anchor_generator: AnchorGenerator, anchor_sampler: AnchorSampler,
+                 feat_sym, anchor_generator=AnchorGenerator, anchor_sampler=AnchorSampler,
```

Comment:
Can you explain?

Reply:
The type of parameter anchor_generator must by Class AnchorGenerator, so i think it should be `:`, not `=` , i will do changes corresponding